rules/linux/process_creation/proc_creation_lnx_pnscan_binary_cli_pattern.yml

```
@@ -0,0 +1,22 @@
+title: Detection of Pnscan Binary Data Transmission Behavior
+id: 97de11cd-4b67-4abf-9a8b-1020e670aa9e
+status: experimental
+description: Detects command line patterns associated with the use of pnscan for sending binary data across the network. This behavior has been identified in a Linux malware campaign targeting Docker, Apache Hadoop, Redis, and Confluence.
+author: David Burkett (@signalblur)
+date: 2024-04-03
```

Hi,
https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md#date-optional
```suggestion
date: 2024/04/03
```